### PR TITLE
started translation to portuguese

### PR DESCRIPTION
**Pull request by HeitorG**
Started translation to portuguese :smile:

### CODE REVIEW

**Comment by EpokK** [comment]
@HeitorG let me know when I can merge :smile:

File: website/pt-br.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		"imprints": "Imprimir",
+		"privacyPolicy": "Politica de privacidade",
+		"termsAndConditions": "Termos &amp; Condições",
+		"team":"Time",
+		"feedbackForum": "Fórum de Feedback"
+	},
+	"home": {
+		"pageTitle": "Email seguro: ProtonMail é um email criptografado livre.",
+		"loading": "Carregando...",
+		"title": "Email seguro baseado <br>em Switzerland",
```
Comment (vinimdocarmo) [inline website/pt-br.json]: I think it would be more correct the translation `"E-mail Seguro Sediado <br>na Suiça"`

File: website/pt-br.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		"subtitle": "Suas comnunicações seguras com ProtonMail.",
```
Comment (vinimdocarmo) [inline website/pt-br.json]: I think it would be more correct the translation `"Proteja Suas Comunicações com ProtonMail"`

**Comment by vinimdocarmo** [comment]
Hey, @HeitorG. Are you still working on this PR ?

**Comment by HeitorG** [comment]
Hi @vinimdocarmo, Yes!

**Comment by vinimdocarmo** [comment]
Nice, @HeitorG. Maybe I can review it when you finish if you don't mind.

**Comment by HeitorG** [comment]
I'm not having much time to continue, if you want to continue will help a lot! :smile:

**Comment by vinimdocarmo** [comment]
@HeitorG, yeah I think I can help. Which repository are you working on ?

**Comment by HeitorG** [comment]
I think I deleted the repository =/ . I think better open another PR.

**Comment by vinimdocarmo** [comment]
I see. Maybe you should close this PR, so when I finish to translate I'm going to open another one.

**Comment by HeitorG** [comment]
Ok! Thank you for your help! :smiley:

**Comment by vinimdocarmo** [comment]
You're welcome :smile:

**Pull request closed without merging**